config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -1621,7 +1621,7 @@ periodics:
   cluster: k8s-infra-prow-build
   annotations:
     testgrid-tab-name: gce-cos-k8sbeta-reboot
-    testgrid-dashboards: sig-release-1.19-blocking
+    testgrid-dashboards: sig-release-master-blocking
```

Comment:
Why is a job named `beta` on master-blocking?

Reply:
This is a result of alpha and beta releases being cut on `master` now :/
The generation script + `test_config.yaml` would need some refactor.

Reply:
I'm suggesting that refactor should be done as part of this PR, or we should wait until we have 1.20 dashboards to hold the beta jobs.  Adding both beta and master jobs to release-master-blocking isn't appropriate